app/bundles/LeadBundle/Entity/LeadListRepository.php

```diff
@@ -459,34 +502,39 @@ public function getLeadsByList($lists, $args = array())
      */
     public function getListFilterExpr($filters, &$parameters, QueryBuilder $q, $not = false, $leadId = null)
     {
-        static $leadTable;
-
         if (!count($filters)) {
 
             return $q->expr()->andX();
         }
 
-        // Get table columns
-        if (null === $leadTable) {
```

Comment:
Was there a reason this static variable and use was removed? Doctrine will query information_schema table every time this method is called and thus the reason for the static use to ensure that the columns are only queried once. If no reason, let's add $companyTable as static as well and keep the if statement.